recipe/meta.yaml

```diff
@@ -1,37 +1,37 @@
-{% set version = "3.1.2" %}
+{% set version = "3.1.3" %}
 
 package:
   name: cdtime
   version: {{ version }}
 
 source:
-  url: https://github.com/cdat/cdtime/archive/v{{ version }}.tar.gz
-  sha256: e6aa613bb509a67f8dc0050d68ffc86c33099e19cfb2b64374b53131b9646e52
+  url: https://github.com/cdat/cdtime/archive/{{ version }}.tar.gz
```

Comment:
@muryanto1, loooks like the "v" got dropped from the tag.  That's no problem but it might make your life easier if you stick with the same convention as before.

Reply:
i updated it.